### ISSUE TITLE
Only thing preventing me to use this player all the time is lack of thumbnails for youtube links.

### ISSUE THREAD

**Issue by formlessdao** [zaps166/QMPlay2#436]
1.If I add 50-100 links over time the lack of thumbnails generated for every link in the playlist makes it harder to distinguish between the links with just the titles alone if there are hundred of youtube links.
Potplayer is the only player I could find that supports large thumbnails in the playlist.
2.When I add a youtube link in Potplayer the title and duration of video AND thumbnail takes like max 1-2 seconds to load but the current QMPLay2 takes many seconds(even 30 seconds) just to display the title of the youtube video in QMPlyer2 playlist.
3.This also prevents me from switching to Linux because Potplayer as you know is only on Windows OS.

Thank you!

**Comment by zaps166** [comment]
Good idea, but playlist icons are very small. So maybe a tool tip with thumbnail or add large icon mode into the playlist.

**Comment by formlessdao** [comment]
" add large icon mode into the playlist."
Yes please. I'd be so incredibly grateful. I have a workflow of adding batches of youtube links and viewing them later and when i have 200-300 links in the playlist it's much easier to know which ones's which if both the title and large thumbnail is present. 
Like i said it's the only thing preventing me to just move to Linux 100%. Not gaming, not Microsoft Excel but a feature that only exists in Potplayer and nowhere else.

**Comment by zaps166** [comment]
> Not gaming

I play mostly on Linux :smile:

**Comment by formlessdao** [comment]
How difficult do you anticipate implementing this large thumbnail mode is time wise?

**Comment by zaps166** [comment]
Enlarge icons - shouldn't be difficult. But I have to store image from youtube into playlist file as base64.

**Comment by formlessdao** [comment]
> 
> 
> But I have to store image from youtube into playlist file as base64.

Sadly I don't know what that means and if that means that's hard to do/implement.

Can you confirm you will implement it? Just so i know and keep checking for updates related this thumbnail feature.

Also i find it so strange that It was a total accident that I found QMPlay. For a project that is so mature and feature rich I am kind of mad that there's no one really talking about it or recommending it. On Linux side there's mainly VLC and MPV....rarely SMplayer but have never seen or found proper videos of OMPlay features showcased .

**Comment by formlessdao** [comment]
5 months and a half...any update please? Is it truly that hard to introduce thumbnail support in the playlist?

**Comment by zaps166** [comment]
It's just reuse thumbnails from YouTube panel in a playlist, but QMPlay2 architecture is not ready for this and playlist code is very old and bad, it's not very easy, but not very hard, too.
